dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -1068,6 +1068,7 @@ private PreparedResult<PolyValue> implement( AlgRoot root, AlgDataType parameter
         }
 
         final AlgDataType jdbcType = QueryProcessorHelpers.makeStruct( root.alg.getCluster().getTypeFactory(), root.validatedRowType );
+        //TODO DL: get metadata instead of null
```

Comment:
ToDo?

Reply:
I added this comment to mark the next line, as we are not able to provide the `fieldOrigins `information and always use `null`. This is correct for the one current use in this method but can be misleading if the variable is used at another place. I now moved the variable and added a comment.